internal/util/k8sutil/k8sutil.go

```diff
@@ -102,7 +110,11 @@ func FormatOperatorNameDNS1123(name string) string {
 // by removing characters from the beginning of label such that len(label) <= 63.
 func TrimDNS1123Label(label string) string {
```

Comment:
This function assumes `label` is already a DNS1123 label

Reply:
@estroz the string was valid but too long. So when we trimmed it we put a leading hyphen which is INVALID.  For example, this `quay-io-raffaelespazzoli-proactive-node-scaling-operator-bundle-latest` turned into `-raffaelespazzoli-proactive-node-scaling-operator-bundle-latest`which is invalid.

Reply:
Right, there's a bug here that could be solved by adding `FormatOperatorNameDNS1123()` but is inefficient since `label` is already a DNS 1123 label. See below.